client/src/app/+my-account/my-account-video-playlists/my-account-video-playlist-elements.component.html

```diff
@@ -14,7 +14,7 @@
       class="videos" myInfiniteScroller [autoInit]="true" (nearOfBottom)="onNearOfBottom()"
       cdkDropList (cdkDropListDropped)="drop($event)" [dataObservable]="onDataSubject.asObservable()"
     >
-      <div class="video" *ngFor="let playlistElement of playlistElements; trackBy: trackByFn" cdkDrag>
+      <div class="video" *ngFor="let playlistElement of playlistElements; trackBy: trackByFn" cdkDrag [cdkDragStartDelay]="isInSmallView() ? 500 : null">
```

Comment:
I think it would be better to put all the logic into the component and use `isInTouchScreen()` instead of `isInSmallView()`:

```
[cdkDragStartDelay]="getDragStartDelay()"
```

```
getDragStartDelay () {
  if (this.screenService.isInTouchScreen()) return 500

  return null
}

Reply:
done, thanks for the review!

Reply:
What about the `isInTouchScreen`? :)

Reply:
Oh, ok, done. Didn't seen your comment